tests/integration/validatorTests.py

```diff
@@ -134,12 +134,13 @@ def test_schema_optional_field(self):
             "test3": "YES",
             "test4": "1",
             "test5": "1",
+            "afa_generated_unique": ""
```

Comment:
Why did we make these double quotes? I thought we were moving away from those

Reply:
I didn't want to change the rest of the file. I can fix it.